app/grants/templates/grants/card.html

```diff
@@ -51,7 +51,7 @@ <h2 class="grant-item__title font-subheader"><a href="{% url 'grants:details' gr
           </div>
         </div>
         <div class="grant-item__state mt-3">
-          {% if grant %} <!-- TODO: wire up grant state -->
+          {% if grant.active %}
```

Comment:
This works, but if a grant is cancelled, it's going to show as `Completed` for now xD

Reply:
True. Though eventually we do need fidelity on this.